src/sagas/console/index.js

```diff
@@ -58,7 +58,7 @@ export function* downloadVmConsole (action) {
         options = yield getConsoleOptions(getConsoleOptionsAction({ vmId }))
       }
 
-      data = adjustVVFile({ data, options, usbFilter, isSpice })
+      data = adjustVVFile({ data, options, usbAutoshare, usbFilter, isSpice })
```

Comment:
As commented in the PR, `isSpice` isn't being assigned properly.  It would probably be better to detect the type of display/console in `adjustVVFile()` as VNC or SPICE and handle the differences there.